external/buildscripts/build_all_osx.pl

```diff
@@ -1,165 +1,163 @@
-use Cwd;

```

Comment:
Line endings are a mess here. They were Windows line endings, but I think that should be Unix instead.